eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -159,6 +168,9 @@ stages:
               environment: githubio
               dependsOn: PublishPackage
 
+              variables:
```

Comment:
It is worth adding in the integration steps below as well.

Reply:
I have added it as staging variable